mne_bids/mne_bids.py

```diff
@@ -657,17 +613,17 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     # Re-save FIF files to fix the file pointer for files with multiple parts
     # This is WIP, see: https://github.com/mne-tools/mne-python/pull/5470
     if ext in ['.fif']:
-        raw.save(raw_file_bids, overwrite=overwrite)
+        raw.save(fname, overwrite=overwrite)
     # CTF data is saved in a directory
     elif ext == '.ds':
-        sh.copytree(raw_fname, raw_file_bids)
+        sh.copytree(raw_fname, fname)
     # BrainVision is multifile, copy over all of them and fix pointers
     elif ext == '.vhdr':
-        copyfile_brainvision(raw_fname, raw_file_bids)
+        copyfile_brainvision(raw_fname, fname)
```

Comment:
this will not go well together with what you write in the docstring on line 497:
```
The file path of the BIDS compatible raw file. In the case of multifile
        systems (e.g., vhdr, .ds etc.), this is the path to a folder containing
        all the files.
```

See also my comment above. Both `copyfile_brainvision` (a multifile system, always 3 files) and `copyfile_eeglab` (a facultative multifile system ... can be 1 file or 2 files) expect the location of the header file, i.e., `myfile.vhdr` or `myfile.set` ... passing a directory is currently not supported and would maybe also not be a good idea.